supabase/migrations/20230923153903_new_permit.sql

```diff
@@ -0,0 +1,48 @@
+CREATE TABLE IF NOT EXISTS
+  new_permits (
+    id serial primary key,
+    created_at timestamptz not null,
+    updated_at timestamptz not null,
+    network smallserial not null,
+    token char(42) not null,
+    amount numeric not null,
+    nonce numeric not null,
```

Comment:
/ask is numeric bigint in supabase

Reply:
I'm skeptical that this is the optimal type. Are you sure?

Reply:
Yes, I tried all of the other datatype..

bigint in supabase is int8, none of them could take that big a number.. It's either numeric or string